    # Publish all joint states -----------------------------------
    joint_state_controller:
      type: joint_state_controller/JointStateController
      publish_rate: 50

     #Additional controllers for this version------------------------------
    head_aux_1_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint:  head_aux_1_joint
      pid: {p: 3.0, i: 1.0, d: 0.0}
    head_aux_2_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint:  head_aux_2_joint
      pid: {p: 3.0, i: 1.0, d: 0.0}
    head_aux_3_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint:  head_aux_3_joint
      pid: {p: 3.0, i: 1.0, d: 0.0}
    auxs_upperlegM1_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint:  auxs_upperlegM1_joint
      pid: {p: 3.0, i: 1.0, d: 0.0}
    auxs_upperlegM2_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint:  auxs_upperlegM2_joint
      pid: {p: 3.0, i: 1.0, d: 0.0}
    auxs_upperlegM3_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint:  auxs_upperlegM3_joint
      pid: {p: 3.0, i: 1.0, d: 0.0}
    upperlegM1_lowerlegM1_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint: upperlegM1_lowerlegM1_joint
      pid: {p: 3.0, i: 1.0, d: 0.0}
    upperlegM2_lowerlegM2_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint: upperlegM2_lowerlegM2_joint
      pid: {p: 3.0, i: 1.0, d: 0.0}
    upperlegM3_lowerlegM3_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint: upperlegM3_lowerlegM3_joint
      pid: {p: 3.0, i: 1.0, d: 0.0}
    internal_lowerleg1_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint: internal_lowerleg1_joint
      pid: {p: 3.0, i: 1.0, d: 0.0}
    internal_lowerleg2_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint: internal_lowerleg2_joint
      pid: {p: 3.0, i: 1.0, d: 0.0}
    internal_lowerleg3_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint: internal_lowerleg3_joint
      pid: {p: 3.0, i: 1.0, d: 0.0}











